frontend/src/component/admin/projectRoles/ProjectRoleForm/PermissionAccordion/PermissionAccordion.tsx

```diff
@@ -43,12 +43,36 @@ const StyledTitle = styled(StringTruncator)(({ theme }) => ({
     marginRight: theme.spacing(1),
 }));
 
+const getPermissionGroupName = (
+    permission: IPermission
+): [groupName: string, order: number] => {
+    const { name } = permission;
+
+    if (name.endsWith('_PROJECT') || name === 'MOVE_FEATURE_TOGGLE') {
+        return ['Project', 0];
+    }
+    if (name.endsWith('_FEATURE') || name === 'UPDATE_FEATURE_ENVIRONMENT') {
+        return ['Feature toggles', 1];
+    }
+    if (name.endsWith('_VARIANTS')) {
+        return ['Variants', 2];
+    }
+    if (name.endsWith('_STRATEGY')) {
+        return ['Strategies', 3];
+    }
+    if (name.endsWith('_CHANGE_REQUEST')) {
+        return ['Change requests', 4];
+    }
+
+    return ['Other', 5];
+};
```

Comment:
While this works now, it seems to introduce a complexity that can be problematic in the future. For example, what if we added a new permission tomorrow that ended with _STRATEGY, for example: `CHANGE_REQUEST_CONFLICT_RESOLTION_STRATEGY` (just making stuff up here). Now as the creator of a new permission you need to be aware that this naming convention grouping exists in the frontend. Over time it might be hard to manage. Perhaps we should consider a more explicit structure?